binaries/ofs.uio/uio/ofs_uio.py

```diff
@@ -687,67 +714,71 @@ def main():
         uio = UIO(uio, args.bit_size, args.region_index,
                   args.mailbox_cmdcsr)
         if not uio.verify_uio():
-            print('Error: please pass the proper arguments')
+            LOG.error('please pass the proper arguments')
             sys.exit(1)
-        print("*****************************\n")
+        LOG.info("**************OFS.UIO*****************\n")
         uio.open()
         # peek/read csr
         if args.peek is not None:
             value = uio.peek(args.region_index, args.bit_size, args.peek)
-            print('peek({}): {}'.format(hex(args.peek), hex(value)))
+            LOG.info('peek(0x%x): 0x%x', args.peek, value)
+
+        elif args.peek_dump is not None:
+            addr = int(args.peek_dump[0], 16)
+            for i in range(int(args.peek_dump[1], 16)):
+                value = uio.peek(args.region_index, args.bit_size, addr)
+                LOG.info('peek(0x%x): 0x%x', addr, value)
+                addr = addr + (int(args.bit_size/8))
```

Comment:
Sorry, just noticed this one.
```suggestion
                addr = addr + (int(args.bit_size)/8)
```